plugins/inputs/jenkins/README.md

```diff
@@ -39,7 +39,9 @@ This plugin does not require a plugin on jenkins and it makes use of Jenkins API
   ## empty will use default value 10
   # max_subjob_per_layer = 10
 
-  ## Jobs to exclude from gathering
+  ## Jobs to include or exclude from gathering
+  ## When using both lists, job_exclude has priority.
+  # job_include = [ "jobA", "jobB/subjobA/subjobB", "jobC*"]
```

Comment:
The default should be an empty list with the meaning of "include all" as I guess that was the behavior before.